### DISCUSSION TITLE
HAFNIUM MS Exchange Exploit

### DISCUSSION THREAD

**Discussion by rampart51** [Security-Onion-Solutions/securityonion#3278] [General] [answered]
Given the recent news regarding the MS Exchange issue, how does one arm SecurityOnion to detect attacks?

Are Zeek rule additions/updates applied automagically?  Or is there a manual process that needs to be followed?  I've read through the docs but am not clear.

Thanks in advance.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3326

**Reply by rampart51** [reply]
Thank you
